redux/actions/profile.js

```diff
@@ -0,0 +1,38 @@
+import firebase from 'firebase';
+import createTestProfile from '../../Profile';
+
+export const PROFILE_CREATE_SUCCESS = 'PROFILE_CREATE_SUCCESS';
+export const PROFILE_CREATE_FAILURE = 'PROFILE_CREATE_FAILURE';
+
+const profileCreatedSuccessfully = profile => ({
+  type: 'PROFILE_CREATE_SUCCESS',
+  payload: profile,
+});
+
+const profileCreateFailure = error => ({
+  type: 'PROFILE_CREATE_FAILURE',
+  error,
+});
+
+const getAndPersistProfile = () => async dispatch => {
+  // Fetch profile from Hub
```

Comment:
Something temporary until we figure out a profile. Just so we can populate some areas/locations